airbyte-integrations/connectors/source-facebook-marketing/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "3.0.0"
+version = "3.0.1"
```

Comment:
nit: 3.1.0, since this is a breaking change on the Airbyte side.

Reply:
Done